cluster-provision/k8s/fetch-latest-cdi.sh

```diff
@@ -39,7 +39,7 @@ fi
 [ ! -d "$provision_dir" ] && echo "directory $provision_dir does not exist!" && exit 1
 [ ! -d "$provision_dir/manifests" ] && echo "directory $provision_dir/manifests does not exist!" && exit 1
 
-tag_name=$(curl -s -L -f https://api.github.com/repos/kubevirt/containerized-data-importer/releases/latest | jq -r '.tag_name')
+tag_name=$(curl -s -L -f https://api.github.com/repos/kubevirt/containerized-data-importer/releases/111644568 | jq -r '.tag_name')
```

Comment:
I think this should remain as latest ? since the script is called `fetch-latest-cdi.sh`

Reply:
whoops my bad